File: GazeChim.Data/Models/BsffPackagingOperation.cs
using GazeChim.Data.Models.Enums;

namespace GazeChim.Data.Models
{
    public class BsffPackagingOperation
    {
        public BsffOperationCode? Code { get; set; }
        public string? Description { get; set; }
        public DateTime? Date { get; set; }
        public bool? NoTraceability { get; set; }

        public BsffPackagingNextDestination? NextDestination { get; set; }
        public Signature? Signature { get; set; }

    }
}

File: GazeChim/ClientApp/src/app/models/bsff/packaging/bsff-packaging-operation.ts
import { BsffOperationCode } from '../enums/bsff-operation-code';
import { Signature } from '../signature';
import { BsffPackagingNextDestination } from './bsff-packaging-next-destination';

export interface BsffPackagingOperation {
  code?: BsffOperationCode;
  description?: string;
  date?: Date;
  noTraceability?: boolean;
  nextDestination?: BsffPackagingNextDestination;
  signature?: Signature;
}
